newsfragments/1716.feature

```diff
@@ -0,0 +1 @@
+``dials.image_viewer``: Add an option to display in a best-fit "image" coordinate frame.
```

Comment:
Probably worth @ndevenish poking at this a little to get some kind of "snapped to grid" idea in there. I have no inspiration otherwise would do myself. Worth making this readable by the general reader.

Reply:
Indeed, should also now say that the option is the new default